lib/renderer.js

```diff
@@ -51,7 +51,16 @@ class Renderer extends MarkedRenderer {
       }
     }
 
-    let out = `<a href="${encodeURL(href)}"`;
+    let out = '<a href="';
+
+    try {
+      out += encodeURL(href);
+    } catch (e) {
```

Comment:
catch binding is optional in Node 10+, tested with eslint.

Reply:
Yeah. It is available since Node.js 10.0.0.

Might remove all of them later.